editor/blocks/quote/index.js

```diff
@@ -0,0 +1,44 @@
+const Editable = wp.blocks.Editable;
+const { query, html } = wp.blocks.query;
+
+wp.blocks.registerBlock( 'core/quote', {
+	title: 'Quote',
+	icon: 'format-quote',
+	category: 'quote',
+
+	attributes: {
+		value: query( 'blockquote', html( 'p' ) ),
+		citation: html( 'cite,footer' )
```

Comment:
I'm not against changing the `post-content` too. Anyway, this is not blocking for me right now